infrastructure/docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
 
   openhim-console:
     container_name: openhim-console
-    image: jembi/openhim-console:latest
+    image: openhim-console-alpinev1
```

Comment:
This is not a valid image. Please revert

Reply:
sure

Reply:
This has not been reverted

Reply:
same as above